pkg/operation/seed/seed.go

```diff
@@ -1843,3 +1862,37 @@ func cleanupOrphanExposureClassHandlerResources(ctx context.Context, c client.Cl
 
 	return nil
 }
+
+// updatePVCIfStorageNotTheSame updates the PVC if passed storage value is not the same as the current one.
+// If there was a successful update it will retuns true as result.
+// Countion!!!. If the passed storage capacity is less than the current one the existing PCV and its PV will be deleted.
+func updatePVCIfStorageNotTheSame(ctx context.Context, k8sClient client.Client, pvcName string, valueToCompare string) (bool, error) {
+	pv := &corev1.PersistentVolumeClaim{}
+	if err := k8sClient.Get(ctx, kutil.Key(v1beta1constants.GardenNamespace, pvcName), pv); err != nil {
+		return false, client.IgnoreNotFound(err)
+	}
+
+	quantityToCompare := resource.MustParse(valueToCompare)
```

Comment:
Why do you use `MustParse` here (which can panic)? I think, this will be cleaner if @vpnachev type suggestion will be followed